CHANGES.md

```diff
@@ -1,3 +1,24 @@
+## edge-19.1.2
+
+* Controller
+  * Retry support! Introduce an `isRetryable` property to service profiles to
+    enable configuring retries on a per-route basis
+  * Add Daemonset support to Web UI, Grafana, and cli `stat` command (thanks,
+    @zknill!)
+* Web UI
+  * Add "meshed" and "no traffic" badges on the resource detail pages
+  * Fixed `linkerd dashboard` to maintain proxy connection when browser open
+    fails
+  * Fixed JavaScript bundling to avoid serving old versions after upgrade
+* CLI
+  * Add `linkerd uninject` command to remove the Linkerd proxy from a Kubernetes
+    config
```

Comment:
As a separate bullet point, we should also mention that `linkerd inject` now support re-injecting resources that have already been injected.